examples/cli.rs

```diff
@@ -93,6 +93,18 @@ enum Cmd {
     ListGroups,
     #[clap(about = "List contacts")]
     ListContacts,
+    #[clap(about = "List messages")]
+    ListMessages,
+    #[clap(about = "List messages from contact")]
+    ListMessagesFromContact {
```

Comment:
if you apply the comments from above and only end up having two ways of retrieving messages, you can use a neat trick from `clap_derive` and have a single command, where one argument has an additional attribute `conflicts_with` with the other (just be careful that it should be set to `master-key` and not `master_key`).